core/api/tests/test_cp_files.py

```diff
@@ -14,13 +12,6 @@ def _test_file(tmp_path):
     return p
 
 
-@pytest.fixture(name="wrong_test_file")
-def _wrong_test_file(tmp_path):
-    p = tmp_path / "adrian.txt"
-    p.write_text("Asa sunt zilele mele")
-    return p
-
-
 class TestCPFiles:
```

Comment:
do you have a multiple uploads test?

Reply:
Yes, but I use the same file for all tests with different names.